v2/pkg/subscraping/sources/dnsdb/dnsdb.go

```diff
@@ -123,8 +181,27 @@ func (s *Source) AddApiKeys(keys []string) {
 func (s *Source) Statistics() subscraping.Statistics {
 	return subscraping.Statistics{
 		Errors:    s.errors,
-		Results:   s.results,
+		Results:   int(s.results),
 		TimeTaken: s.timeTaken,
 		Skipped:   s.skipped,
 	}
 }
+
+func getOffset(ctx context.Context, session *subscraping.Session, headers map[string]string) (rate, error) {
```

Comment:
The conflation here between "rates" and "offsets" (in our particular case, only the MAXIMUM offset value being of interest) is confusing. I think the code should ONLY make reference to this max offset value. The work done by

`strconv.ParseUint(rate.OffsetMax.String()
`
can also probably be folded into this function.

Reply:
Alright addressed this with: https://github.com/kelvinatorr/subfinder/pull/1/commits/df86f0fe59d579baf7af775cacbbdfddcfbc8d8f